NYPLCardCreator/Classes/UserCredentialsViewController.swift

```diff
@@ -114,6 +114,8 @@ final class UserCredentialsViewController: TableViewController {
     
     self.tableView.estimatedRowHeight = 120
     self.tableView.allowsSelection = false
+    
+    NotificationCenter.default.addObserver(self, selector: #selector(self.endSignIn), name: Notification.Name.init("NYPLSettingsAccountsSignInFinishedNotification"), object: nil)
```

Comment:
A good point that you had brought up before. From our discussion, I've decided to remove this UX functionality to better separate what's going on between the Creator and the host app. Sign in will still commence from the main app once the card has been created.